.github/workflows/ci.yml

```diff
@@ -289,8 +351,6 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
-        # nightly required for testing until this issue is resolved:
```

Comment:
Please leave this comment in unless you expect that the underlying issue will never be resolved.

In general, we want to avoid using nightly Rust for build processes because they are unpredictable. (See diffs at line 391.)

Reply:
Sorry I did not mean to delete this.